static/gsApp/components/superuser/superuserWarning.tsx

```diff
@@ -101,29 +99,8 @@ function SuperuserWarning({organization}: Props) {
       >
         Superuser
       </Tooltip>
-    </SuperuserBadge>
+    </Badge>
   );
 }
 
 export default SuperuserWarning;
-
-const SuperuserBadge = styled(Badge)<{stackedNav: boolean}>`
```

Comment:
While adding the superuser badge to the mobile nav, I realized that the existing styles weren't going to work and it would be better add these styles where rendered instead. So the styles are removed from this component and added at the call site instead with a wrapper div.